[29-Jul-2015 13:50:13 Europe/Moscow] PHP Fatal error:  Uncaught exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'zf2guard'@'localhost' (using password: YES)' in /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php:253
Stack trace:
#0 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php(253): PDO->__construct('mysql:dbname=zf...', 'zf2guard', 'zf2guard', Array)
#1 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Pdo.php(249): Zend\Db\Adapter\Driver\Pdo\Connection->connect()
#2 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Sql/Sql.php(128): Zend\Db\Adapter\Driver\Pdo\Pdo->createStatement()
#3 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/TableGateway/AbstractTableGateway.php(237): Zend\Db\Sql\Sql->prepareStatementForSqlObject(Object(Zend\Db\Sql\Select))
#4 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/TableGateway/AbstractTableGateway.php(208): Zend\Db\TableGateway\AbstractTableGateway->executeSelect(Object(Zend\Db\Sql\Select))
#5 in /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php on line 264
[29-Jul-2015 13:50:20 Europe/Moscow] PHP Fatal error:  Uncaught exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'zf2guard'@'localhost' (using password: YES)' in /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php:253
Stack trace:
#0 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php(253): PDO->__construct('mysql:dbname=zf...', 'zf2guard', 'zf2guard', Array)
#1 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Pdo.php(249): Zend\Db\Adapter\Driver\Pdo\Connection->connect()
#2 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Sql/Sql.php(128): Zend\Db\Adapter\Driver\Pdo\Pdo->createStatement()
#3 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/TableGateway/AbstractTableGateway.php(237): Zend\Db\Sql\Sql->prepareStatementForSqlObject(Object(Zend\Db\Sql\Select))
#4 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/TableGateway/AbstractTableGateway.php(208): Zend\Db\TableGateway\AbstractTableGateway->executeSelect(Object(Zend\Db\Sql\Select))
#5 in /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php on line 264
[29-Jul-2015 13:50:20 Europe/Moscow] PHP Fatal error:  Uncaught exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'zf2guard'@'localhost' (using password: YES)' in /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php:253
Stack trace:
#0 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php(253): PDO->__construct('mysql:dbname=zf...', 'zf2guard', 'zf2guard', Array)
#1 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Pdo.php(249): Zend\Db\Adapter\Driver\Pdo\Connection->connect()
#2 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Adapter.php(211): Zend\Db\Adapter\Driver\Pdo\Pdo->createStatement('?            SE...')
#3 /var/www/Zf2Guard/module/Application/src/Application/Model/Pages.php(41): Zend\Db\Adapter\Adapter->createStatement('?            SE...', NULL)
#4 /var/www/Zf2Guard/module/Application/src/Application/Model/Pages.php(35): Application\Model\Pages->FetchAll(Object(BjyProfiler\Db\Adapter\ProfilingAdapter), '?            SE...')
# in /var/www/Zf2Guard/vendor/twig/twig/lib/Twig/Template.php on line 337
[29-Jul-2015 13:53:06 Europe/Moscow] PHP Fatal error:  Uncaught exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'zf2guard'@'localhost' (using password: YES)' in /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php:253
Stack trace:
#0 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php(253): PDO->__construct('mysql:dbname=zf...', 'zf2guard', 'zf2guard', Array)
#1 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Pdo.php(249): Zend\Db\Adapter\Driver\Pdo\Connection->connect()
#2 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Sql/Sql.php(128): Zend\Db\Adapter\Driver\Pdo\Pdo->createStatement()
#3 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/TableGateway/AbstractTableGateway.php(237): Zend\Db\Sql\Sql->prepareStatementForSqlObject(Object(Zend\Db\Sql\Select))
#4 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/TableGateway/AbstractTableGateway.php(208): Zend\Db\TableGateway\AbstractTableGateway->executeSelect(Object(Zend\Db\Sql\Select))
#5 in /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php on line 264
[29-Jul-2015 13:59:00 Europe/Moscow] PHP Fatal error:  Uncaught exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'zf2guard'@'localhost' (using password: YES)' in /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php:253
Stack trace:
#0 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php(253): PDO->__construct('mysql:dbname=zf...', 'zf2guard', 'zf2guard', Array)
#1 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Pdo.php(249): Zend\Db\Adapter\Driver\Pdo\Connection->connect()
#2 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Sql/Sql.php(128): Zend\Db\Adapter\Driver\Pdo\Pdo->createStatement()
#3 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/TableGateway/AbstractTableGateway.php(237): Zend\Db\Sql\Sql->prepareStatementForSqlObject(Object(Zend\Db\Sql\Select))
#4 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/TableGateway/AbstractTableGateway.php(208): Zend\Db\TableGateway\AbstractTableGateway->executeSelect(Object(Zend\Db\Sql\Select))
#5 in /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php on line 264
[29-Jul-2015 13:59:01 Europe/Moscow] PHP Fatal error:  Uncaught exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'zf2guard'@'localhost' (using password: YES)' in /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php:253
Stack trace:
#0 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php(253): PDO->__construct('mysql:dbname=zf...', 'zf2guard', 'zf2guard', Array)
#1 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Pdo.php(249): Zend\Db\Adapter\Driver\Pdo\Connection->connect()
#2 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Sql/Sql.php(128): Zend\Db\Adapter\Driver\Pdo\Pdo->createStatement()
#3 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/TableGateway/AbstractTableGateway.php(237): Zend\Db\Sql\Sql->prepareStatementForSqlObject(Object(Zend\Db\Sql\Select))
#4 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/TableGateway/AbstractTableGateway.php(208): Zend\Db\TableGateway\AbstractTableGateway->executeSelect(Object(Zend\Db\Sql\Select))
#5 in /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php on line 264
[29-Jul-2015 14:00:47 Europe/Moscow] PHP Fatal error:  Uncaught exception 'PDOException' with message 'SQLSTATE[28000] [1045] Access denied for user 'zf2guard'@'localhost' (using password: YES)' in /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php:253
Stack trace:
#0 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php(253): PDO->__construct('mysql:dbname=zf...', 'zf2guard', 'zf2guard', Array)
#1 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Pdo.php(249): Zend\Db\Adapter\Driver\Pdo\Connection->connect()
#2 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Sql/Sql.php(128): Zend\Db\Adapter\Driver\Pdo\Pdo->createStatement()
#3 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/TableGateway/AbstractTableGateway.php(237): Zend\Db\Sql\Sql->prepareStatementForSqlObject(Object(Zend\Db\Sql\Select))
#4 /var/www/Zf2Guard/vendor/zendframework/zend-db/src/TableGateway/AbstractTableGateway.php(208): Zend\Db\TableGateway\AbstractTableGateway->executeSelect(Object(Zend\Db\Sql\Select))
#5 in /var/www/Zf2Guard/vendor/zendframework/zend-db/src/Adapter/Driver/Pdo/Connection.php on line 264
